backend/validateUser_test.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"backend/endpoints"
+	"database/sql"
+	"net/http"
+	"backend/encryption"
+	"net/http/httptest"
+	"testing"
+	"encoding/json"
+
+	"github.com/gin-gonic/gin"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestValidateUser tests the validateUser function
+func TestValidateUser(t *testing.T) {
+	// Set up test database connection
+	var err error
+	db, err = sql.Open("sqlite3", "shows.db")
+	if err != nil {
+		t.Fatalf("Failed to connect to database: %v", err)
+	}
+	defer db.Close()
+
+	// Clean up existing users
+	_, err = db.Exec("DELETE FROM Users")
```

Comment:
same thing as getAllUsers_test.go, I have a problem with clearing the entire table just for the test, as that could clear real user data if used on the production server.